### PR TITLE
Updated code of SCF Assignment 2 with JUnit Test

### PR DESCRIPTION
**Pull request by meta-chirag-jain**

### CODE REVIEW

File: com/metacube/SCF/Assignment2/HexCalc.java [hunk 1]
```diff
@@ -0,0 +1,154 @@
+package GET2018.com.metacube.SCF.Assignment2;
+
+public class HexCalc {
```
Comment (pallavisoni) [inline com/metacube/SCF/Assignment2/HexCalc.java]: Add comment at the top of the class specifying purpose, author and copyrights info. Class name should be more meaningful.

File: com/metacube/SCF/Assignment2/HexCalc.java [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+	
+	private int decValue1, decValue2, answer;
+	private String digits  = "0123456789ABCDEF";
+	private String result;
+
```
Comment (pallavisoni) [inline com/metacube/SCF/Assignment2/HexCalc.java]: Class level variables are needed only when they are to be initialized from constructor. In your case we can simply create local variables. So avoid using class level variables until unless really necessary.

File: com/metacube/SCF/Assignment2/HexCalcTest.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+		
+		answer = operation.convertDecToHex(289);
+		assertEquals("121", answer);
+		
+		result = operation.convertHexToDec("121");
+		assertEquals(289, result);
+		
+		result = operation.convertHexToDec("A");
+		assertEquals(10, result);
+	}
```
Comment (pallavisoni) [inline com/metacube/SCF/Assignment2/HexCalcTest.java]: Add a test scenario to check for blank spaces.

File: com/metacube/SCF/Assignment2/JobScheduler.java [hunk 4]
```diff
@@ -0,0 +1,110 @@
+package GET2018.com.metacube.SCF.Assignment2;
+
+public class JobScheduler {
```
[the remark this answers is not in the document]
Comment (meta-chirag-jain) [reply com/metacube/SCF/Assignment2/JobScheduler.java]: done

**Pull request merged**